server/moodle/mod/livequiz/amd/src/quizcreator.js

```diff
@@ -0,0 +1,196 @@
+
+define(['jquery'], function ($) {
+    return {
+        init: function () {
+            $(document).ready(function () { // Ensure DOM is fully loaded
+                console.log("Make quiz JS loaded");
+
+                open_question_creation_modal();
+            });
+        }
+    };
+    function open_question_creation_modal() {
+        // Corrected button ID
+        let add_question_button = document.getElementById("id_buttonaddquestion");
+
+            add_question_button.addEventListener('click', () => {create_question_modal()});
+
+    }
+
+    function create_question_modal() {
+        let modal_div = document.createElement("div"); // Sådan laver man en div.
```

Comment:
sørg for at ensrette kommentarer så alle kommentarer er på engelsk

Reply:
Jeg har slettet den kommentar nu.